starter-code/css/main.css

```diff
@@ -6,3 +6,175 @@ Green: #00B172
 White: #FFF
 
 */
+
+
+body{
+  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.nav{
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100px;
+  background-color: white;
+  z-index: 10;
+  }
+
+#logo {
+  height:60px;
+  float: left;
+  margin-left: 5%;
+  margin-top: 1%;
+  padding-bottom: 1%;
+}
+
+#logo:hover {
+  animation-name: logo;
+  animation-duration: 0.2s;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+}
+
+@keyframes logo {
+  0% {margin-top: 0%;}
+  100% {margin-top: 1%;}
+}
+
+.nav-list li{
+  float: right;
+  list-style: none;
+  display: inline;
+  font-size: 1.5em;
+  vertical-align: text-top;
+  padding-left: 2%;
+  margin-top: 1%;
+}
+
+.nav-list{
+  margin-right: 5%;
+}
+
+.main-image{
+  background-image: url("../images/landing.jpg");
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
Try `background-size: cover` with `background-position: center` and see if you like that better when resizing the window.